packages/app/src/definitions.ts

```diff
@@ -11,6 +11,21 @@ export interface FirebaseAppPlugin {
    * @since 0.1.0
    */
   getOptions(): Promise<GetOptionsResult>;
+
+  /**
+   * Initialize a new Firebase App with the provided name and Firebase project configuration
+   * @param options
+   */
+  initializeAppWithConfig(options: FirebaseConfigOptions): Promise<void>;
+
+  /**
+   * Check if a Firebase App with the provided name is initialized
+   * @param options
+   * @returns Promise object with property boolean "result", true if initialized
+   */
+  appIsInitialized(
+    options: FirebaseAppName,
+  ): Promise<FirebaseAppInitializedResult>;
```

Comment:
```suggestion
  /**
   * Initialize a new Firebase App with the provided name and Firebase project configuration.
   *
   * @since 6.2.0
   */
  initializeApp(options: InitializeAppOptions): Promise<void>;
  /**
   * Get all initialized FIrebase apps.
   *
   * @since 6.2.0
   */
  getApps(): Promise<GetAppsResult>;
```

I always try to stick to the official SDK for the method names/etc.